src/Item/DuplicateItem/DuplicateItem.js

```diff
@@ -42,8 +42,8 @@ const DuplicateItem = ({
   const location = useLocation();
 
   const { isLoading: isInstanceLoading, instance } = useInstanceQuery(instanceId);
-  const { isLoading: isHoldingLoading, holding } = useHoldingQuery(holdingId);
-  const { isLoading: isItemLoading, item } = useItemQuery(itemId);
+  const { isLoading: isHoldingLoading, holding } = useHoldingQuery(holdingId, { tenantId: location?.state?.tenantTo });
+  const { isLoading: isItemLoading, item } = useItemQuery(itemId, { tenant: location?.state?.tenantTo });
```

Comment:
The competing naming of `tenant` vs `tenantId` is historical and there's nothing you can do about that, but it really smacks you in the face here. 

I am strongly inclined to say that `useHoldingQuery` (ugh, bad names everywhere, should be `useHoldingsRecordQuery` 😭) should be refactored so the keys in its options-argument match those in `useItemQuery`, i.e. `{ tenant }`.